### ISSUE TITLE
Fix  `declaration-property-value-no-unknown` reported ranges for multiple math functions

### ISSUE THREAD

**Issue by romainmenke** [stylelint/stylelint#9053]
### What minimal example or steps are needed to reproduce the bug?

```css
a {
  margin-inline: calc(2px + 1px) calc(1px + 1);
}
```

### What minimal configuration is needed to reproduce the bug?

```js
export default {
  rules: {
    "declaration-property-value-no-unknown": true
  }
};

```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICGACYAdAdilBbJATgOYCWmAtGQDZkzwphJVgAUATAA4AeKA1CgEZuASgZNWQnvwHCA3FgC+IADQgAZiSowAcklxxEMLno5aAdGADOlleAiYNRBCCMcIBAC4pYapAFcqLwxsFAIAmEt6YJwcdBBYMCpCJA8Se3IOAggOGE8AT3IANyY-GHJMCHI-TABrCoB3TDj6DzCYLBwFRXlMW0gHEiIAMXd8D2dLDzytGkwPC3tHM1wAKxtVWA4bRGiUOMnpmFmPZr2QAQB2MzYzAAY45Q6zg5myD3J+x3JJpEwoQigpziABZbnc7nFFFgQAogA)

### Which Stylelint-related dependencies are you using?

```json
{
  "stylelint": "17.2.0",
  "stylelint-config-standard": "40.0.0"
}
```

### What did you expect to happen?

I expected the second expression to be indicated

### What actually happened?

The first was indicated, even when this is correct.

<img width="401" height="139" alt="Image" src="https://github.com/user-attachments/assets/56885704-0cb6-404d-b3e5-fe73e11f055b" />
